iFixit/react-commerce
# Refactor product list children

connects #1361

This PR is updating the productListChildren layout to align to the  new product list page described in https://github.com/iFixit/design/issues/510

### QA

1. Visit [Vercel preview](https://react-commerce-git-refactor-product-list-children-ifixit.vercel.app/Parts)
2. Verify that the product list children adhere to the new design.
